.github/workflows/build-integration.yml

```diff
@@ -0,0 +1,88 @@
+name: gocardless/stolon-pgbouncer/build-integration
+on:
+  push:
+env:
+  DOCKER_PASS: xxxx9e14
+  DOCKER_USER: xxxxdmin
+  GITHUB_TOKEN: xxxx6d08
+jobs:
+  unit-integration:
+    defaults:
+      run:
+        working-directory: "/go/src/github.com/gocardless/stolon-pgbouncer"
+    runs-on: ubuntu-latest
+    container:
+      image: gocardless/stolon-pgbouncer-circleci:2020050701
+      env:
+        PGHOST: 127.0.0.1
+        PGUSER: postgres
+    steps:
+    - uses: actions/checkout@v2
+    - name: Compile ginkgo test suites
+      run: ginkgo build -r -race .
+    - name: Run unit tests
+      run: find pkg -type f -name '*.test' -printf "%h %f\n" | xargs -n2 sh -c 'cd $0 && su postgres -c ./$1'
+  build:
+    defaults:
+      run:
+        working-directory: "/go/src/github.com/gocardless/stolon-pgbouncer"
+    runs-on: ubuntu-latest
+    container:
+      image: gocardless/stolon-pgbouncer-circleci:2020050701
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build test binaries
+      run: make linux
+    - uses: actions/upload-artifact@v2
+      with:
+        path: |-
+          /go/src/github.com/gocardless/stolon-pgbouncer/bin/stolon-pgbouncer.linux_amd64
+          /go/src/github.com/gocardless/stolon-pgbouncer/bin/stolon-pgbouncer-acceptance.linux_amd64
```

Comment:
I think uploading and downloading the artifact here and below is unnecessary. We could probably just bundle the build & acceptance into one job?